src/containers/pages/InterventionPlan/IRS/plan/index.tsx

```diff
@@ -2008,6 +2116,27 @@ class IrsPlan extends React.Component<
                 .catch(() => {
                   this.setState({ isSaveDraftDisabled: false });
                 });
+
+              // POST to retire unassigned assignments
+              if (retiredAssignments.length) {
+                await OpenSrpAssignmentService.create([...retiredAssignments])
+                  .then(res => {
+                    // todo - hook in success notification
+                  })
+                  .catch(err => {
+                    // todo - handle errors
+                  });
+              }
+              // POST to create new/updated assignments
+              if (nextAssignments.length) {
+                await OpenSrpAssignmentService.create([...nextAssignments])
+                  .then(res => {
+                    // todo - hook in success notification
+                  })
+                  .catch(err => {
+                    // todo - handle errors
+                  });
+              }
```

Comment:
NOTE - when POST'ing to the `assignLocationsAndPlans` endpoint, the payload array may only contain either retired assignments OR active assignments.